modules/core/arrow-test/src/main/kotlin/arrow/test/generators/LiftGen.kt

```diff
@@ -0,0 +1,54 @@
+package arrow.test.generators
+
+import arrow.Kind
+import arrow.core.ForId
+import arrow.core.ForListK
+import arrow.core.ForNonEmptyList
+import arrow.core.ForOption
+import arrow.core.ForSequenceK
+import arrow.core.Id
+import arrow.core.ListK
+import arrow.core.NonEmptyList
+import arrow.core.Option
+import arrow.core.SequenceK
+import io.kotlintest.properties.Gen
+
+interface LiftGen<F> {
+  /**
+   * lifts a Gen<A> to the context F. the resulting Gen can be used to create types Kind<F, A>
+   */
+  fun <A> liftGen(gen: Gen<A>): Gen<Kind<F, A>>
+}
+
+fun Option.Companion.liftGen() = object : LiftGen<ForOption> {
```

Comment:
Is there a reason these cannot be done with `@extension`? For tests I don't really care as much, so up to you :)

Reply:
no, i didn't try that yet. wanted to get feedback and review first before investing more time in that approach. handwriting the instances was my easiest first solution.

Reply:
i have changed that. the GenK instances are now generated using `@extension`

Reply:
i also cleaned up all Semialign related Laws (and related tests) to use GenK now.